application/src/main/java/net/furizon/backend/infrastructure/pretix/PretixPagingUtil.java

```diff
@@ -18,6 +18,7 @@ public static <U, P extends PretixPaging<U>> void fetchAll(
         while (hasNext) {
             final P response = callable.apply(currentPage);
             if (response.getResults() == null) {
+                //TODO: Are we really sure we want to continue? Won't this cause infinite loops?
                 continue;
```

Comment:
if result is null, there is no way we have more pages, so we don't need to process the loop and can brake it tbh

Reply:
So even for you it's safe to change to a `break;` instruction?

Reply:
I think so, let's do hasNext = false and break; the loop